### DISCUSSION TITLE
Modify Card width

### DISCUSSION THREAD

**Discussion by Spartacus68** [kinghat/tabbed-card#101] [General]
Hi,
I would like to discuss the following topic:

I have tried to modify the card-width of this example below, but I havn´t found a solution to combine the settings with 

```
card_mod:
  style: |
    ha-card {
       width: 70%;   
       height: 150px;
     }
```
I think it is worth to have it within the styling properties or did I miss anything?

**EXAMPLE:**
```
type: custom:mod-card
card_mod:
  style:
    tabbed-card $:
      mwc-tab:nth-child(1) $: |
        .mdc-tab__text-label {
          font-size: 12px;
          white-space: pre;
        }
      mwc-tab:nth-child(1) ha-icon $: |
        ha-svg-icon {
          color: green;
        }
card:
  type: custom:tabbed-card
  attributes:
    icon: mdi:circle
  tabs:
    - card:
        type: vertical-stack
        cards:
          - type: markdown
            content: Prgramm 1
            title: Rolladen Programm 1
      attributes:
        icon: mdi:numeric-1-circle
        label: Tab 1
    - card:
        type: vertical-stack
        cards:
          - type: markdown
            content: Prgramm 2
            title: Rolladen Programm 2
      attributes:
        icon: mdi:numeric-2-circle
        label: Tab 2
    - card:
        type: vertical-stack
        cards:
          - type: markdown
            content: Programm 3
            title: Rolladen Programm 3
      attributes:
        icon: mdi:numeric-3-circle
        label: Tab 3
```

**Comment by ildar170975** [comment]
Do you mean a "width of a whole card"?
If yes - all Lovelace cards has an automatically defined width dependently on a current layout.

If you mean a "width of some tab" - then your styling (1st post) is wrong:
```
card_mod:
  style: |
    ha-card {
       width: 70%;   
       height: 150px;
     }
```
Tab's width is set dependently on a tab's title:
![изображение](https://user-images.githubusercontent.com/71872483/235965550-3b3df3e2-36ac-4659-a9d9-c80025fb2dbd.png)
But you may try to change a width by card-mod.

Also, why have you posted another card-mod example with changing a font-size & icon color since it is not related to changing a width?

**Comment by Spartacus68** [comment]
Hi,
no I really mean the card width and it can be modified when using 2 cards in a horizontal stack (see example with markdown and swipe-card). I used this for the wipe card with 70% and  the markdown uses the remaining 30%. If you do not modify the card-width, both cards will use the sapce in ratio 1:1



![image](https://user-images.githubusercontent.com/121975453/235974676-de5ca46e-0fd4-4f31-802a-bab56e2488c5.png)

**Reply by ildar170975** [reply]
None of lovelace cards has an option to define a card's width.
Do not think that tabbed-card could be an exception.
Your only way is using card-mod.
